src/utils.js

```diff
@@ -0,0 +1,19 @@
+// subset of Object.assign
+export function assign(object, ...hashes) {
+  for (let i = 0; i < hashes.length; i++) {
+    let hash = hashes[i];
+    for (let key in hash) {
+      object[key] = hash[key];
+    }
+  }
+  return object;
+}
+
+// Array.prototype.some
+export function some(array, ...rest) {
+  return array.filter.apply(array, rest).length > 0;
```

Comment:
Does the babel polyfill not use an early return? I.e. what if the first element is true, in a list of 1000 elements?